### DISCUSSION TITLE
error during training

### DISCUSSION THREAD

**Discussion by buhtigntt** [bmaltais/kohya_ss#1158] [General]
Hello everyone, I encountered this error during training (attached image). Could you please help me resolve this issue? Thank you very much in advance.
![kohya](https://github.com/bmaltais/kohya_ss/assets/126443636/598d71cb-3933-4148-808d-f7e706c0fd55)

**Comment by mats4d** [comment]
well, I updated Xforrmers to Xformer 21 pre-release and it works now, not sure if this is the actual cause of the issue, but seems to be

https://pypi.org/project/xformers/0.0.21.dev565/
